src/include/daos_task.h

```diff
@@ -72,15 +74,15 @@ typedef enum {
 	DAOS_OPC_CONT_DESTROY_SNAP,
 
 	/** Transaction APIs */
-	DAOS_OPC_TX_OPEN = 37,
+	DAOS_OPC_TX_OPEN = 39,
 	DAOS_OPC_TX_COMMIT,
 	DAOS_OPC_TX_ABORT,
 	DAOS_OPC_TX_OPEN_SNAP,
 	DAOS_OPC_TX_CLOSE,
 	DAOS_OPC_TX_RESTART,
 
 	/** Object APIs */
-	DAOS_OPC_OBJ_QUERY_CLASS = 44,
+	DAOS_OPC_OBJ_QUERY_CLASS = 45,
```

Comment:
It should start from 46, that may explain why you hit the assert.

Reply:
ok will update, still unsure why it needs to be 46?